src/parser/shaman/shared/constants.js

```diff
@@ -0,0 +1,13 @@
+import SPELLS from 'common/SPELLS';
+
+export const HEALING_ABILITIES_AMPED_BY_EARTH_SHIELD = [
```

Comment:
Thanks for catching that, you're indeed correct that many of the spells in my list don't work with earth shield as they're not shaman specific. There are a few spells missing however:

```suggestion
export const HEALING_ABILITIES_AMPED_BY_EARTH_SHIELD = [
	SPELLS.HEALING_WAVE.id,
	SPELLS.ASCENDANCE_INITIAL_HEAL.id,
	SPELLS.WELLSPRING_HEAL.id,
	SPELLS.UNLEASH_LIFE_TALENT.id,
```
ascendance redistribution and cbt technically do as well by proxy; but they're such special cases that i'll look at it at some other time

Reply:
Oh, I totally missed the "see more" button on the spell data in wowhead page I linked. I'll fix that up.